covid-19.md

```diff
@@ -1,13 +1,18 @@
 ---
 layout: blog
 title: COVID-19 Updates
-date: 2021-12-08
+date: 2022-03-31
 ---
 
 We will keep this page updated with new information as it becomes available.
 
 Last updated: {{ page.date | date: "%e %B %Y" }}.
 
```

Comment:
Should we have a title or something here, so that it doesn't just launch straight into the list?
Even just `## Events` might do?

Reply:
I'll add some prose

Reply:
I changed most of the wording really...